.github/vale/styles/Klaw/branding.yml

```diff
@@ -22,3 +22,5 @@ swap:
   "node[ -]?js?": NodeJS
   "type[ -]?scripts?": TypeScript
   "(?i)ssl": SSL
+  "(?i)klaw": Klaw
```

Comment:
Can you remove this rule? It's listed in `branding-warning-temp.yml`, where it belongs (for now). It will cause the pipeline to fail, too, because this is not fixed yet.